### PR TITLE
sentry...

### PR DESCRIPTION
**Pull request by myniqx**
image

### CODE REVIEW

**Review by 7alip — APPROVED** [review]
Thanks

**Comment by myniqx** [comment]
@7alip when i check the details, i realize i shouldnt use sentry() function. i ll remove them too. and use that strapi function directly.
image

File: apps/api/src/utils/sentry.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const sentry = error => {
```
[the remark this answers is not in the document]
Comment (7alip) [reply apps/api/src/utils/sentry.ts:1]: @myniqx What about when you use the instance, they mention in the docs

```ts
const sentryService = strapi.plugin('sentry').service('sentry');
```

```ts
sentryService.sendError(error)
```

[the remark this answers is not in the document]
Comment (myniqx) [reply apps/api/src/utils/sentry.ts:1]: do you mean: get instance from sentry.ts then add sendError function all the place sentry function used ?

[the remark this answers is not in the document]
Comment (7alip) [reply apps/api/src/utils/sentry.ts:1]: Actually it doesn't matter if you call `sentryService.sendError(error)` or `strapi.plugin('sentry').service('sentry').sendError(error)` when you need it.

**Pull request merged**